scripts/s3uploading.js

```diff
@@ -0,0 +1,116 @@
+const shell = require("shelljs");
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+
+const workdir = process.env.GITHUB_WORKSPACE;
+const s3 = new AWS.S3({
+    "AWS_ACCESS_KEY_ID": process.env.AWS_ACCESS_KEY_ID,
+    "AWS_SECRET_ACCESS_KEY": process.env.AWS_SECRET_ACCESS_KEY
+});
+const tmp = `${workdir}/tmp`;
+const readmeRegex = new RegExp('(readme\.md|readme\.org)', 'i');
```

Comment:
There can also be just `README` and `.rst` is also supported by GitHub, AFAICT.

Reply:
added support of README and .rst